src/olympia/versions/models.py

```diff
@@ -236,23 +245,17 @@ def get_due_date_reason_q_objects(cls):
                 needshumanreview__is_active=True,
                 needshumanreview__reason=NeedsHumanReview.REASONS.ADDON_REVIEW_APPEAL,
             ),
-            'has_developer_reply': Q(
-                needshumanreview__is_active=True,
-                needshumanreview__reason=NeedsHumanReview.REASONS.DEVELOPER_REPLY,
-            ),
             # Themes are special as noted above.
             'is_from_theme_awaiting_review': is_from_theme_awaiting_review,
-            # Everything else shares the is_other_needs_human_review condition.
-            'needs_human_review_auto_approval_disabled': Q(
-                is_other_needs_human_review,
-                needshumanreview__reason=NeedsHumanReview.REASONS.AUTO_APPROVAL_DISABLED,
+            'needs_human_review_from_cinder_forwarded_abuse': Q(
+                needshumanreview__is_active=True,
+                needshumanreview__reason__in={
+                    NeedsHumanReview.REASONS.CINDER_ESCALATION,
+                },
```

Comment:
this should have been changed to
```python
                needshumanreview__reason=NeedsHumanReview.REASONS.CINDER_ESCALATION,
```
I'll fix in the PR for https://github.com/mozilla/addons/issues/15413

Reply:
Ah thanks. It's just a code-quality issue, right? The code should work either way? At least I did not see any errors in tests or during manual testing.